Release 0.2.2

ff137 opened the pull request:
My way of asking for a quick patch release, please :-)

I know there is probably another release being planned after #19 is merged, but please in the meantime can we get a patch release, that allows for the latest askar version.

This will help untangle a dependency knot for our acapy fork (where we have to remove the didwebvh feature, in favour of having the latest askar, so we can use the pagination-ordering feature from https://github.com/openwallet-foundation/acapy/pull/3173. This release is also a requirement to wrap up that PR)

andrewwhitehead commented:
Due to the renaming PR the main branch actually implements "did:webvh:0.4" which isn't really a thing. I think the method might need to be changed back to `tdw` for a compatible release.

ff137 commented:
Oh my ... so, essentially I would need to revert #16?

I think a good strategy is for the main branch to always be releasable, and a development branch to be used for work that is in progress.

Hmm... so I think the simplest would be to create `development` from `main` as it is now, and then click "Revert" at the bottom of #16 to remove it from `main`. That way it's migrated to a new branch and main is releasable again.

Sound good? I believe only an admin can create new branches.

andrewwhitehead commented:
I've created a new 0.2.x branch from just before #16. I think it would be easier to do the release from there instead of un-merging and re-merging on the main branch.

ff137 commented:
Awesome, that's even better. I'll rebase 👍

ff137 commented:
Cool - so I've just taken 0.2.x, and allowed any askar version >=0.3.2. Updated the lock file, and bumped the version in pyproject.toml to 0.2.2. 
(Also removed the `[project] requires-python = ">=3.10"` setting at the top, because it was highlighted red)

andrewwhitehead commented:
Because the publishing ability for the did-tdw project was removed for this repo (switched over to the new project name) I had to publish the package manually, but it should be available now.

jamshale commented:
This shouldn't have failed publishing to `did_webvh`. I don't really know why it failed. 

Even though `did:webvh:0.4` I don't really want to revert the naming in acapy. I don't think it's a big deal to have it this way.

The fail is odd. It says either the publisher wasn't setup right, (Even though it worked previously). Or there was an internal error with github oy pypi.

`This generally indicates a trusted publisher configuration error, but could also indicate an internal error on GitHub or PyPI's part`

Maybe we could try running the pypi publish action again or doing it manually?

andrewwhitehead commented:
@jamshale It failed because it was publishing a new version of `did_tdw`, the old package name. I expect publishing under the new package name would work. ACA-Py should either pick up the new `did_webvh` package implementing 0.5, or can use this package for testing 0.4.
